src/physics/PhysicsList.cc

```diff
@@ -33,6 +40,55 @@ void PhysicsList::ConstructProcess() {
   AddParameterization();
   Shielding::ConstructProcess();
   ConstructOpticalProcesses();
+  EnableThermalNeutronScattering();
+}
+
+// Add thermal neutron scattering to this physics list.
+// Based on the technique shown here: http://tinyurl.com/hszjtln
+// See http://tinyurl.com/zx53kjq for more details.
+void PhysicsList::EnableThermalNeutronScattering() {
+
+  // Get the particle definition for neutrons
+  G4ParticleDefinition* n_definition = G4Neutron::Definition();
+
+  // Get the elastic scattering process used for neutrons
+  G4HadronicProcess* n_elastic_process = nullptr;
+  G4ProcessVector* proc_vec = n_definition->GetProcessManager()
+    ->GetProcessList();
+  for (size_t i = 0; i < proc_vec->size(); ++i) {
```

Comment:
This is a type mismatch (won't compile for me) -- should be `int i` since G4ProcessVector::size returns G4int. Also, prefer postfix `++` for native types.